Wrappers/Python/cil/utilities/dataexample.py

```diff
@@ -71,14 +64,20 @@ def download_data(cls, data_dir):
 
         '''
         if os.path.isdir(os.path.join(data_dir, cls.FOLDER)):
-            print("Dataset already exists in " + data_dir)
+            print("Dataset folder already exists in " + data_dir)
         else:
-            if input("Are you sure you want to download " + cls.FILE_SIZE + " dataset from " + cls.URL + " ? (y/n)") == "y":
-                print('Downloading dataset from ' + cls.URL)
-                cls._download_and_extract_from_url(os.path.join(data_dir,cls.FOLDER))
-                print('Download complete')
+            user_input = input("Are you sure you want to download {cls.ZIP_FILE} dataset from Zenodo record {cls.ZENODO_RECORD}? [Y/n]: ") if prompt else 'y'
+            if  user_input.lower() == 'y':
+                zenodo_get([cls.ZENODO_RECORD, '-g', cls.ZIP_FILE, '-o', data_dir])
+
+                # unzip file
+                with ZipFile(os.path.join(data_dir, cls.ZIP_FILE), 'r') as zip_ref:
+                    zip_ref.extractall(os.path.join(data_dir, cls.FOLDER))
+                os.remove(os.path.join(data_dir, cls.ZIP_FILE))
             else:
                 print('Download cancelled')
+                return False
+            return True
```

Comment:
```suggestion
            if user_input.lower() not in ('y', 'yes'):
                print('Download cancelled')
                return False

            zenodo_get([cls.ZENODO_RECORD, '-g', cls.ZIP_FILE, '-o', data_dir])
            with ZipFile(os.path.join(data_dir, cls.ZIP_FILE), 'r') as zip_ref:
                zip_ref.extractall(os.path.join(data_dir, cls.FOLDER))
            os.remove(os.path.join(data_dir, cls.ZIP_FILE))
            return True
```

slight refactor to remove most indents...